commit.go

```diff
@@ -249,7 +249,8 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 	}
 	if exporting {
 		// Copy everything.
-		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter))
+		// TOM: todo add credsContext
```

Comment:
Removed.  Not needed before merging.  This will be turned on when the cred params are put into the 'buildah commit' command in the next PR for buildah auth.